libfreerdp/core/surface.c

```diff
@@ -221,17 +222,31 @@ static BOOL update_write_surfcmd_bitmap_ex(wStream* s, const TS_BITMAP_DATA_EX*
 	return TRUE;
 }
 
-BOOL update_write_surfcmd_surface_bits(wStream* s, const SURFACE_BITS_COMMAND* cmd)
+BOOL update_write_surfcmd_surface_bits(wStream* s, const SURFACE_BITS_COMMAND* ccmd)
 {
+	SURFACE_BITS_COMMAND cmd = *ccmd;
```

Comment:
Why copy the whole struct if we only every override cmdType?

Reply:
true, should not be necessary.

Reply:
@mfleisz done, only `cmdType` copied.